app/views/shared/_cdn_assets.html.erb

```diff
@@ -0,0 +1,8 @@
+<meta charset="utf-8">
+
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.bundle.min.js"></script>
```

Comment:
CDN links are very simple to set up, but there is a better way to load Bootstrap, especially the JS dependencies. You can read about some of the reasons in [this article](https://shkspr.mobi/blog/2020/10/please-stop-using-cdns-for-external-javascript-libraries/). There is also [this installation guide](https://www.timdisab.com/installing-bootstrap-4-on-rails-6/) for another way to install bootstrap and it's dependencies.